src/app/settings/change-password.component.ts

```diff
@@ -77,6 +82,17 @@ export class ChangePasswordComponent implements OnInit {
 
         const strengthResult = this.passwordGenerationService.passwordStrength(this.newMasterPassword,
             this.getPasswordStrengthUserInput());
+
+        if (this.enforcedPolicyOptions != null &&
+            !(this.policyService.evaluateMasterPassword(
```

Comment:
same. dont need extra parens

Reply:
Updated - only had them in there for my sanity :D